OracleSOASuite/dockerfiles/README.md

```diff
@@ -1,20 +1,201 @@
+    RCUPREFIX=SOA1
+    DB_PASSWORD=Welcome1
+    DB_SCHEMA_PASSWORD=Welcome1
+    ADMIN_PASSWORD=Welcome1
+    MANAGED_SERVER=soa_server1
+    DOMAIN_TYPE=soa
+    
+To start a docker container with a SOA domain and the WebLogic AdminServer call `docker run command` and pass the above `adminserver.env.list` file.
+
+A sample docker run command is given below:
+
+     $docker run -i -t  --name soaas --network=SOANet -p 7001:7001  -v /scratch/DockerVolume/SOAVolume/SOA:/u01/oracle/user_projects   --env-file ./adminserver.env.list oracle/soa:12.2.1.3
+     
+The options "-i -t" in the above command runs the container in interactive mode and you will be able to see the commands running in the container. This includes the command for RCU creation, domain creation and configuration followed by starting the Admin Server. 
+
+IMPORTANT: You need to wait till all the above commands are run before you can access the AdminServer Web Console.
+The following lines highlight when the Admin server is ready to be used:    
+
+INFO: Admin server is running
+INFO: Admin server running, ready to start managed server
+
+The above line indicate the admin server started successfully with the name `soaas`, mapping container port 7001 to host port 7001 enables accessing of the Weblogic host outside of the local host, connecting to SOANet network enables accessing the DB container by it's name i.e soadb. This includes the command to tail logs to keep the container up and running.
+
+
+## Creating a container for SOA Server
+
+Start a container to launch the Managed Server from the image created. The environment variables used to run the Managed Server image are defined in the file soaserver.env.list. 
+
+Create an environment file soaserver.env.list with the below details:
+
+    MANAGED_SERVER=<managed server name, For Exp:- soa_server1, osb_server1, bpm_server1>
+    DOMAIN_TYPE=<soa/osb/bpm>
+    ADMIN_HOST=<Admin host name>
+    ADMIN_PORT=<port number where Admin Server is running>
+    
+Sample Data will look like this:
+
+    MANAGED_SERVER=soa_server1
+    DOMAIN_TYPE=soa
+    ADMIN_HOST=slc09cwi.us.oracle.com
+    ADMIN_PORT=7001
+    
+To start a docker container for SOA server you can simply call `docker run` command and passing `soaserver.env.list`. 
+
+A sample docker run command is given below:
+
+    $ docker run -i -t  --name soams --network=SOANet -p 8001:8001   --volumes-from soaas   --env-file ./soaserver.env.list oracle/soa:12.2.1.3 "/u01/oracle/dockertools/startMS.sh"
+    
+Using --volumes-from reuses the volume created by the Admin container. In the above `docker run` command, `soaas` is the name of the Admin server container started in the previous step and we must use the same name used for the AdminServer to start the SOA managed server. 
+
+This includes the command to tail logs to keep the container up and running.
+
+IMPORTANT: You need to wait till all the above commands are run before you can start the SOA Server.
```

Comment:
Another location you could use a block quote to call attention to this important note.